pkg/kubelet/rkt/rkt.go

```diff
@@ -754,9 +755,15 @@ func podFinishedMarkerPath(podDir string, rktUID string) string {
 	return filepath.Join(podDir, "finished-"+rktUID)
 }
 
-func podFinishedMarkCommand(touchPath, podDir, rktUID string) string {
-	// TODO, if the path has a `'` character in it, this breaks.
-	return touchPath + " " + podFinishedMarkerPath(podDir, rktUID)
+func podFinishedMarkCommand(touchPath, podDir, rktUID string) (string, error) {
+	apostropheFlag, err := regexp.MatchString(`'`, touchPath)
```

Comment:
`strings.ContainsRune` is much faster

Reply:
I have changed the method from `regexp.MatchString` to `strings.ContainsRune` :)

Reply:
@tallclair plz take a look :)